proto/pulumi/resource.proto

```diff
@@ -238,3 +240,15 @@ message TransformResponse {
     google.protobuf.Struct properties = 1; // the transformed input properties.
     TransformResourceOptions options = 2; // the options for the resource.
 }
+
+message RegisterProviderRequest {
```

Comment:
I might be missing something here, but don't we need to send the parameters for parameterized providers here as well?  Or is that all encoded in the checksums?

Reply:
We do need to send the parameters. But we haven't yet designed that fully yet.